tests/integration/models/application.spec.coffee

```diff
@@ -581,3 +581,58 @@ describe 'Application Model', ->
 					.then =>
 						promise = balena.models.application.isTrackingLatestRelease(@application.id)
 						m.chai.expect(promise).to.eventually.be.true
+
+	describe 'given a multicontainer application with a single offline device', ->
+
+		givenMulticontainerApplication()
+
+		itShouldBeAnApplicationWIthDeviceServiceDetails = (application) ->
+			m.chai.expect(_.omit(application, 'owns__device', 'commit')).to.deep.equal(_.omit(@application, 'owns__device', 'commit'))
+			m.chai.expect(application.owns__device).to.have.lengthOf(1)
+			m.chai.expect(application.owns__device[0]).to.deep.match
+				device_name: @device.device_name
+				uuid: @device.uuid
+				is_on__commit: @currentRelease.commit
+				current_services:
+					web: [
+						id: @newWebInstall.id
+						service_id: @webService.id
+						image_id: @newWebImage.id
+						commit: 'new-release-commit'
+						status: 'downloading'
+						download_progress: 50
+					,
+						id: @oldWebInstall.id
+						service_id: @webService.id
+						image_id: @oldWebImage.id
+						commit: 'old-release-commit'
+						status: 'running'
+						download_progress: 100
+					]
+					db: [
+						id: @newDbInstall.id
+						service_id: @dbService.id
+						image_id: @newDbImage.id
+						commit: 'new-release-commit'
+						status: 'running'
+						download_progress: 100
+					]
+
+            # Should filter out deleted image installs
```

Comment:
Let's fix this mixed indentation before merging.

Reply:
That's the thing I was talking about, see how it looks in my editor:
<img width="965" alt="screenshot 2019-01-10 at 16 56 19" src="https://user-images.githubusercontent.com/2152815/50980327-be613f80-14f8-11e9-97e1-b65272083241.png">
Just removing all indentation and adding again fixed it in other places. Will do it here too.